This produces warnings during testing.  Since we do not care much about
coverage we simply remove it.

--- lz4-4.3.3/tox.ini.orig
+++ lz4-4.3.3/tox.ini
@@ -9,7 +9,7 @@
 #     PYTHONMALLOCSTATS = 'yes'
 usedevelop = True
 commands =
-    pytest --cov=lz4/block --cov=lz4/frame --tb=long {posargs} tests/block tests/frame
+    pytest --tb=long {posargs} tests/block tests/frame
 
 [pytest]
 addopts = -x --tb=long --showlocals
